[2025-08-16T02:11:50.770+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-16T02:11:50.810+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline_dvc.train_fraud_detector manual__2025-08-16T02:11:03.537932+00:00 [queued]>
[2025-08-16T02:11:50.825+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline_dvc.train_fraud_detector manual__2025-08-16T02:11:03.537932+00:00 [queued]>
[2025-08-16T02:11:50.826+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-16T02:11:50.852+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): train_fraud_detector> on 2025-08-16 02:11:03.537932+00:00
[2025-08-16T02:11:50.862+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9655) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-16T02:11:50.865+0000] {standard_task_runner.py:63} INFO - Started process 9672 to run task
[2025-08-16T02:11:50.865+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline_dvc', 'train_fraud_detector', 'manual__2025-08-16T02:11:03.537932+00:00', '--job-id', '280', '--raw', '--subdir', 'DAGS_FOLDER/train_pipeline_dag.py', '--cfg-path', '/tmp/tmpa95zwf1e']
[2025-08-16T02:11:50.867+0000] {standard_task_runner.py:91} INFO - Job 280: Subtask train_fraud_detector
[2025-08-16T02:11:50.934+0000] {task_command.py:426} INFO - Running <TaskInstance: ml_pipeline_dvc.train_fraud_detector manual__2025-08-16T02:11:03.537932+00:00 [running]> on host 952e994ef77e
[2025-08-16T02:11:51.069+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='yas' AIRFLOW_CTX_DAG_ID='ml_pipeline_dvc' AIRFLOW_CTX_TASK_ID='train_fraud_detector' AIRFLOW_CTX_EXECUTION_DATE='2025-08-16T02:11:03.537932+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-16T02:11:03.537932+00:00'
[2025-08-16T02:11:51.071+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-16T02:11:51.093+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-16T02:11:51.095+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n            cd /opt/***/ml_pipeline\n            dvc pull\n            dvc repro train_model\n            dvc push\n        ']
[2025-08-16T02:11:51.108+0000] {subprocess.py:86} INFO - Output:
[2025-08-16T02:11:54.505+0000] {subprocess.py:93} INFO - Everything is up to date.
[2025-08-16T02:11:57.781+0000] {subprocess.py:93} INFO - 'data/raw.dvc' didn't change, skipping
[2025-08-16T02:11:57.837+0000] {subprocess.py:93} INFO - Stage 'preprocess' didn't change, skipping
[2025-08-16T02:11:57.879+0000] {subprocess.py:93} INFO - Stage 'train_model' didn't change, skipping
[2025-08-16T02:11:57.886+0000] {subprocess.py:93} INFO - Data and pipelines are up to date.
[2025-08-16T02:12:01.720+0000] {subprocess.py:93} INFO - Everything is up to date.
[2025-08-16T02:12:02.137+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-16T02:12:02.139+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-16T02:12:02.191+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ml_pipeline_dvc, task_id=train_fraud_detector, run_id=manual__2025-08-16T02:11:03.537932+00:00, execution_date=20250816T021103, start_date=20250816T021150, end_date=20250816T021202
[2025-08-16T02:12:02.252+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-16T02:12:02.279+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-16T02:12:02.281+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
